source/index.js

```diff
@@ -195,7 +196,8 @@ module.exports = async (input = 'patch', options) => {
 			}
 		]);
 
-		if (!options.exists && !pkg.private) {
+		const isExternalRegistry = npm.isExternalRegistry(pkg);
+		if (!options.exists && !(pkg.private || isExternalRegistry)) {
```

Comment:
Isn't this just the same as?

```
if (!options.exists && !pkg.private && !isExternalRegistry) {
```

Feels more readable imo :)

Reply:
done